File: agentpress_cli/_exit_codes.py
"""Centralized exit codes. Mirrors bin/lib/exit_codes.js so Node + Python
CLIs have identical contracts."""

OK = 0
ERRORS_FOUND = 1          # validation errors, unknown command, user abort
STRICT_OR_INTERNAL = 2    # strict-mode warning escalation OR internal error
FILE_NOT_FOUND = 3        # agents.txt missing where expected

File: bin/lib/exit_codes.js
'use strict';

// Centralised exit codes. Used by every verb module so the CLI's
// contract with shell users / CI is consistent.

module.exports = Object.freeze({
  OK: 0,
  ERRORS_FOUND: 1,          // generic non-success: validation errors, unknown command, user abort
  STRICT_OR_INTERNAL: 2,    // strict-mode warning escalation OR internal/unexpected error
  FILE_NOT_FOUND: 3,        // agents.txt missing where one was expected
});
